packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/selection/SelectionControllerIntegrationTests.java

```diff
@@ -129,8 +135,10 @@ public void beforeEach() {
     @Sql(scripts = { "/scripts/cleanup.sql" }, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
     public void givenSemanticObjectWhenRequestingSelectionDescriptionThenTheSelectionDescriptionIsSent() {
         String representationId = "selectionDialog://?representationDescription=" + URLEncoder.encode(this.selectionDescriptionProvider.getSelectionDialogDescriptionId(), StandardCharsets.UTF_8);
-        Map<String, Object> variables = Map.of("editingContextId", PapayaIdentifiers.PAPAYA_PROJECT.toString(), "representationId", representationId, "targetObjectId",
-                PapayaIdentifiers.PROJECT_OBJECT.toString());
+        List<SelectionDialogVariable> selectionDialogVariables = List.of(new SelectionDialogVariable("targetObjectId", PapayaIdentifiers.PROJECT_OBJECT.toString()),
```

Comment:
It's not very easy to read, I'll make some space

Reply:
agree